Add feed listing for publishers

webmasterjunkie opened the pull request:
Shows how to get all available product feeds.

webmasterjunkie committed:
Add feed listing for publishers

Shows how to get all available product feeds.

pete001 commented on src/common.apib:
```diff
@@ -166,3 +166,17 @@
++ campaign_feeds_id: `1011l10`
++ name: `UK Destinations` - The name of the product feed
++ filename: `uk_destinations.csv`
++ location: (string) - Where the file is located
```
a colon is used, but no example value is given

pete001 commented on src/common.apib:
```diff
@@ -166,3 +166,17 @@
++ campaign_id: `10l176`
++ title: `SportBetting` - The name of the campaign
++ creative_id: `300441`
++ location_compressed: (string) - Where the compressed file is located
```
as above

webmasterjunkie committed:
Add examples

https://github.com/PerformanceHorizonGroup/apidocs/pull/17#discussion-di
ff-70069049
https://github.com/PerformanceHorizonGroup/apidocs/pull/17#discussion-di
ff-70069073

pete001 commented on src/apiary.apib:
```diff
@@ -1392,6 +1392,29 @@ View all Creative for the specified Campaign.
     + Attributes
         + creatives (array[Creative Wrapper])
 
+## List all Available Feeds [GET /user/publisher/{publisher_id}/feed{?campaign_id,active}]
```
This is the method for listing all feeds, can you also request an individual feed by its id? This is a recurring pattern in the docs, because it allows you to define the entity and have a specific dredd test against the entity attributes.

webmasterjunkie replied:
You can't request a specific feed here. It only allows you to filter by campaign and feed status - at least that I have found.
